mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1449,7 +1468,16 @@
     <!-- Preference option for never saving passwords in Fenix -->
     <string name="preferences_passwords_save_logins_never_save">Never save</string>
     <!-- Preference for autofilling saved logins in Fenix -->
-    <string name="preferences_passwords_autofill">Autofill</string>
+    <string name="preferences_passwords_autofill" tools:ignore="UnusedResources">Autofill</string>
```

Comment:
Can we just delete this string if it's not in use anymore?

Reply:
@Amejia481 : Can you perhaps give guidance on this? I believe we usually delete those, looking at past PRs and our current strings in repo